modules/nf-core/genmap/index/main.nf

```diff
@@ -1,29 +1,44 @@
 process GENMAP_INDEX {
-    tag "$fasta"
-    label 'process_high'
+    tag "$meta.id"
```

Comment:
Is meta data is required for every module now?

Reply:
Not necessarily but this module has reference data as input, which we are trying to add meta to. It's also more user friendly to be able to pass some meta along with the files.